libgalois/src/analytics/bfs/bfs.cpp

```diff
@@ -543,6 +492,9 @@ katana::analytics::BfsStatistics::Compute(
   BfsImplementation::Graph graph = pg_result.value();
 
   uint32_t source_node = std::numeric_limits<uint32_t>::max();
+  // TODO(lhc) max dist should be the last node
+  //           for now, keep it since it requires python modification.
+  //           The number of visited nodes is enough for verification.
```

Comment:
The stats need to be updated for the new information format. Are we still computing distances here? Or should we compute different stats?

Updating the Python wrappers is not hard.